docs/usage.md

```diff
@@ -201,7 +201,7 @@ const sequelize = new Sequelize('sqlite:relativePath/dbname.db')
 
 ### PostgreSQL
 
-The library for PostgreSQL is`pg@~3.6.0` You'll just need to define the dialect:
+The library for PostgreSQL is`pg@~6.1.0` You'll just need to define the dialect:
```

Comment:
I would say the latter, there is a high chance we won't support 8.0 when it comes out